src/views/includes/interior-navigation/base/link.blade.php

```diff
@@ -0,0 +1,10 @@
+<a
+    href="{{ data_get($link, 'href', '#') }}"
+    @if(data_get($link, 'active', false))
+        class="px-2 border-l-2 transition-colors text-gray-900 font-bold border-red-600"
+    @else
+        class="px-2 border-l-2 transition-colors border-transparent hover:text-gray-900"
+    @endif
+>
+    {{ data_get($link, 'text', 'No text') }}
```

Comment:
Same thing here. Let's throw an error if text is not provided.

Reply:
Done. 👍